src/Nethermind/Nethermind.Evm/EOF/EvmObjectFormat.cs

```diff
@@ -286,10 +288,90 @@ bool ValidateInstructions(ReadOnlySpan<byte> container, ref EofHeader? header)
                     header = null; return false;
                 }
 
+                if (_releaseSpec.IsEip4200Enabled)
+                {
+                    if (opcode is Instruction.RJUMP or Instruction.RJUMPI)
+                    {
+                        if (i + 2 > sectionSize)
+                        {
+                            if (_loggingEnabled)
+                            {
+                                _logger.Trace($"EIP-4200 : Static Relative Jump Argument underflow");
+                            }
+                            header = null; return false;
```

Comment:
I put the breakpoint in this line and run 4200 tests. I haven't hit the breakpoint which means that it is not covered by unit tests